include/seqan3/search/fm_index/fm_index_cursor.hpp

```diff
@@ -267,11 +261,13 @@ class fm_index_cursor
      *
      * No-throw guarantee.
      */
-    template <Alphabet char_t>
+    template <typename char_t>
```

Comment:
```suggestion
    template <std::ConvertibleTo<typename index_t::char_type> char_t>
```

Reply:
We usually prefer doing statis_asserts, no?